### PR TITLE
userguide: hyperscan information updated

### PR DESCRIPTION
**Pull request by jmtaylor90**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://suricata.readthedocs.io/en/latest/devguide/codebase/contributing/contribution-process.html
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

Describe changes:
- The distributions I checked all seem to have hyperscan available as a package
- If it was necessary to install from source I was thinking it would be good to consolidate the instructions under advanced installation documentation
-

#suricata-verify-pr:
#suricata-verify-repo:
#suricata-verify-branch:
#suricata-update-pr:
#suricata-update-repo:
#suricata-update-branch:
#libhtp-pr:
#libhtp-repo:
#libhtp-branch:

### CODE REVIEW

File: doc/userguide/performance/hyperscan.rst [hunk 1]
```diff
@@ -4,133 +4,29 @@ Hyperscan
 Introduction
 ~~~~~~~~~~~~
 
-"Hyperscan is a high-performance multiple regex matching library." https://www.hyperscan.io
+"Hyperscan is a high performance regular expression matching library..." https://www.intel.com/content/www/us/en/developer/articles/technical/introduction-to-hyperscan.html
```
Comment (jufajardini) [inline doc/userguide/performance/hyperscan.rst:7]: Suggestion:
```diff
- "Hyperscan is a high performance regular expression matching library..." https://www.intel.com/content/www/us/en/developer/articles/technical/introduction-to-hyperscan.html
+ "Hyperscan is a high performance regular expression matching library (...)" (cf. 
+ https://www.intel.com/content/www/us/en/developer/articles/technical/introduction-to-hyperscan.html)
```

**Review by jufajardini — CHANGES_REQUESTED** [review]
Adding a suggestion on top of what Victor has indicated.

Thinking that having the removed steps in an `advanced` section makes sense, too.

File: doc/userguide/performance/hyperscan.rst (hunk 1 above)
Comment (jmtaylor90) [reply doc/userguide/performance/hyperscan.rst:7]: That was my thinking as well, do we want to do the advanced installation instructions as part of this PR?

Comment (jufajardini) [reply doc/userguide/performance/hyperscan.rst:7]: If that's ok with you, this is good, as we ensure that we won't remove them and not bring them back.
But we can also just make sure that these two changes go into the same release, IMHO, and then they don't have to be part of the same PR.

**Comment by jmtaylor90** [comment]
> I think it would be nice to mention how to install in ubuntu/debian/fedora/centos?
> 
> Also might be good to add a note about vectorscan being a drop in replacement for ARM

I am wondering where you all think the best location for the hyperscan installation information would be? Do we want to put it in the performance/hyperscan.rst file or put all installation (hyperscan and suricata) steps into the general install page?

**Comment by jufajardini** [comment]
> > I think it would be nice to mention how to install in ubuntu/debian/fedora/centos?
> > Also might be good to add a note about vectorscan being a drop in replacement for ARM
> 
> I am wondering where you all think the best location for the hyperscan installation information would be? Do we want to put it in the performance/hyperscan.rst file or put all installation (hyperscan and suricata) steps into the general install page?

I'd keep it under hyperscan, as that's something optional...

Sorry for not answering earlier!

**Comment by jufajardini** [comment]
Hello @jmtaylor90 thanks for the time you contribute to Suri :)

Can you still work on this? Did my answers clarify your questions, or do you need more input?

**Comment by jmtaylor90** [comment]
> Hello @jmtaylor90 thanks for the time you contribute to Suri :)
> 
> Can you still work on this? Did my answers clarify your questions, or do you need more input?

I apparently missed the response from March but yes, I will rework this with the comments/suggestions from you and Victor. Thanks!

**Comment by catenacyber** [comment]
friendly ping @jmtaylor90 in case you forgot about this ;-)

**Comment by jmtaylor90** [comment]
> friendly ping @jmtaylor90 in case you forgot about this ;-)

I have a draft branch that I need to get back to. I should have something in the next week or so if that works?

**Comment by catenacyber** [comment]
> I have a draft branch that I need to get back to. I should have something in the next week or so if that works?

That would be perfect, thanks

**Comment by jmtaylor90** [comment]
continued in #9202

**Pull request closed without merging**